### ISSUE TITLE
legion_prof.py error when running with -s option

### ISSUE THREAD

**Issue by cyang49** [StanfordLegion/legion#150]
Tested using master branch commit id: 994360363e2b80ee80cad58e4f7e010fc9924f7f
========================================================================

```
work-pc:~/projects/legion/examples/full_circuit$ ./ckt_sim -ll:cpu 4 -hl:prof 1 -logfile prof_%.log
work-pc:~/projects/legion/examples/full_circuit$ $LG_RT_DIR/../tools/legion_prof.py -s prof_0.log
Reading log file prof_0.log...
WARNING: Skipped 6 lines in prof_0.log
Matched 1212 lines
****************************************************
   TASK STATS
****************************************************
Traceback (most recent call last):
  File "/home/chihchiehy/projects/legion/runtime/../tools/legion_prof.py", line 2160, in <module>
    main()
  File "/home/chihchiehy/projects/legion/runtime/../tools/legion_prof.py", line 2147, in main
    state.print_stats(verbose) 
  File "/home/chihchiehy/projects/legion/runtime/../tools/legion_prof.py", line 1911, in print_stats
    self.print_task_stats(verbose)
  File "/home/chihchiehy/projects/legion/runtime/../tools/legion_prof.py", line 1902, in print_task_stats
    proc.update_task_stats(stat)
  File "/home/chihchiehy/projects/legion/runtime/../tools/legion_prof.py", line 558, in update_task_stats
    task.update_task_stats(stat, self)
TypeError: update_task_stats() takes exactly 2 arguments (3 given)
```
[prof_0.log.tar.gz](https://github.com/StanfordLegion/legion/files/372211/prof_0.log.tar.gz)

**Comment by elliottslaughter** [comment]
Could you attach the log file that caused the problem? Thanks.

**Comment by cyang49** [comment]
Attached to the original post

**Comment by lightsighter** [comment]
Fix pushed to master branch with commit 4aed646
